elastalert/elastalert.py

```diff
@@ -704,7 +704,10 @@ def get_ch_data(self, rule, starttime, endtime, agg_key, freshquery,aggregation)
                     "aggregations":[aggregation]
                 }
         try:
-            res = requests.post(self.query_endpoint, json=data)
+            headers = {}
```

Comment:
I see that this will be called during error rate alone? What happens to _msearch calls?